x-pack/plugins/infra/public/apps/common_providers.tsx

```diff
@@ -12,7 +12,7 @@ import {
   KibanaContextProvider,
 } from '../../../../../src/plugins/kibana_react/public';
 import { TriggersActionsProvider } from '../utils/triggers_actions_context';
-import { ClientPluginDeps } from '../types';
```

Comment:
I changed some names here to try to make it all consistent. We now have most of our types prefaced with "InfraClient" (eventually to be LogsClient and MetricsClient, with "Client" to distinguish from the "Server" types), and then we use StartDeps/StartExports and SetupDeps/SetupExports to specify the dependencies and exports coming and going out of each lifecycle method.